feed/tests/test_feed_views.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from feed.models import Post
+
+
+USERNAME = 'username'
+USER_PASS = 'secret_pass'
+POST_DETAIL_URL = '/post/'
+
+
+@pytest.mark.django_db
+class TestPostDetailView:
+    def test_detail_view_page_entrypoint(self, client):
+        # Testing to see if a valid post id gets a valid detail view page
+        post = Post.posts.main_feed().first()
```

Comment:
In case there are no posts, main_feed().first() is throwing an exception?

Reply:
Thanks, didn't thought on that.
Should I assert False in the case where there are no posts exist?
Is it even something that I should handle? because we are running those tests under development mode and not in production so that there is always the data migration running first.
The questions aren’t for you @amittcohen specifically, but for whoever knows.
Maybe a better way would be to create a post as a fixture and that will make sure there is always at least one post available

Reply:
I have decided to add a fixture that saves post to DB, that making sure there is at least one post exist in DB at all times.